Detour/Include/DetourNavMeshQuery.h

```diff
@@ -194,6 +194,24 @@ class dtNavMeshQuery
 					  const dtQueryFilter* filter,
 					  dtPolyRef* path, int* pathCount, const int maxPath) const;
 
+	/// Finds a path from the start polygon to the closest (by geodesic distance) to any of the goals.
```

Comment:
This is just by normal euclidean distance.

Reply:
It's by geodesic distance.  The normal proof that A* returns the shortest path by geodesic distance also holds for the multi-start case.

Reply:
Ok, I understand now. I was not familiar with this term in this context. I thought this was related to sphere geometry.

If it confuses me it will probably also confuse users, so perhaps leave it out? Though I understand what you are trying to convey (but with the remark below, it seems to be not even true in the formal sense?)

Reply:
Yeah, it is a term from graph theory.

I will try to reword that comment so the point is made that its closest by shortest path distance without the technical terminology -- something like "Finds the path from the start polygon to _any_ of the goal polygons.  The shortest shortest path is returned."

Reply:
If we limit ourselves to only traversing bidirectional edges it is true in the formal sense :)

Reply:
> I will try to reword that comment so the point is made that its closest by shortest path distance without the technical terminology -- something like "Finds the path from the start polygon to _any_ of the goal polygons. The shortest shortest path is returned."

The "shortest shortest path" is a bit funny. Maybe something like: "Finds the shortest path ending at any of the specified locations"?

> If we limit ourselves to only traversing bidirectional edges it is true in the formal sense :)

Still, it may not be immediately obvious that it is by geodesic distance on the "virtual" graph induced by the filter when also applying this restriction you mention. But anyway this is a digression! :-)